index.html

```diff
@@ -0,0 +1,325 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Cahee</title>
+    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@300;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header class="header-bock">
+      <div class="container">
+        <div class="header__wrapper">
+          <div name="home" class="header__top">
```

Comment:
Тут якщо ти хотів зробити навігацію за якорями, то потрібен атрибут `id`, а тегу `<a>` даєш атрибут `href="#home"`. Те ж стосується і далі секцій: атрибут `name` використовується в інпутах.
```suggestion
          <div id="home" class="header__top">
```